### PR TITLE
Fix padding and rtl

### PR DESCRIPTION
**Pull request by muffinresearch**
Relates to #4243 (logged out part)

# Before:

screen shot 2017-01-10 at 20 02 54-fullpage

# After: 

screen shot 2017-01-10 at 20 00 40-fullpage

screen shot 2017-01-10 at 20 01 02-fullpage

screen shot 2017-01-10 at 20 01 21-fullpage

### CODE REVIEW

File: static/css/devhub/new-landing/common.less [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
       max-width: @grid-max;
-      padding: 0;
+      padding: 0 20px;
     }
   }
 
   p {
     color: @color-text;
-    font-weith: 300;
+    font-weight: 300;
```
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/common.less]: heh, nice find!

File: static/css/devhub/new-landing/common.less [hunk 2]
```diff
@@ -119,9 +130,15 @@
     height: 275px;
     position: absolute;
     right: -186px;
-    top: 100px;
+    top: ~"calc(50% - 57px)"; // Top of the image to the cog.
```
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/common.less]: "the the cog" - what's a cog?

File: static/css/devhub/new-landing/common.less [hunk 3]
```diff
@@ -152,9 +196,17 @@
   background-repeat: no-repeat;
   background-position: 50% 10px;
 
+  html[dir=rtl] & {
+    background-position: 50% 10px;
```
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/common.less]: isn't that identical to the default, non rtl version? Shouldn't that be reversed?

**Comment by EnTeQuAk** [comment]
Just a few nits/questions, r+wc

File: static/css/devhub/new-landing/common.less (hunk 3 above)
Comment (muffinresearch) [inline static/css/devhub/new-landing/common.less]: Hmm yeah, good catch that appears to be redundant.

File: static/css/devhub/new-landing/common.less (hunk 2 above)
Comment (muffinresearch) [inline static/css/devhub/new-landing/common.less]: The cog is this:



So I'm referring to the top of the image to the cog at the top of the ladder.

Comment (muffinresearch) [inline static/css/devhub/new-landing/common.less]: I'll try and clarify the comment a bit more.

**Pull request merged**